extensions/mssql/src/objectExplorerNodeProvider/hdfsProvider.ts

```diff
@@ -165,7 +165,7 @@ export class FolderNode extends HdfsFileSourceNode {
 			nodeStatus: undefined,
 			nodeType: this._nodeType,
 			nodeSubType: undefined,
-			iconType: 'Folder'
+			iconType: this._nodeType === Constants.MssqlClusterItems.Connection ? 'HDFSFolder' : 'Folder'
```

Comment:
Could we just override the `getNodeInfo` method in ConnectionNode and change the iconType there instead? It'd be nicer to encapsulate logic for the HDFS node there rather than putting in the parent class.